nephos/fabric/peer.py

```diff
@@ -61,72 +61,73 @@ def setup_peer(opts, upgrade=False):
         upgrade (bool): Do we upgrade the deployment? False by default.
         
     """
-    peer_namespace = get_namespace(opts, opts["peers"]["msp"])
-    for release in get_peers(opts=opts):
-        # Deploy the CouchDB instances
-        version = get_version(opts, "hlf-couchdb")
-        config_yaml = f'{opts["core"]["dir_values"]}/hlf-couchdb/cdb-{release}.yaml'
-        if not upgrade:
-            extra_vars = helm_extra_vars(version=version, config_yaml=config_yaml)
-            helm_install(
-                opts["core"]["chart_repo"],
-                "hlf-couchdb",
-                f"cdb-{release}",
-                peer_namespace,
-                extra_vars=extra_vars,
-                
-            )
-        else:
-            preserve = (
-                HelmPreserve(
+    for msp in get_msps(opts=opts):
+        peer_namespace = get_namespace(opts, msp=msp)
+        for release in get_peers(opts=opts, msp=msp):
+            # Deploy the CouchDB instances
+            version = get_version(opts, "hlf-couchdb")
+            config_yaml = f'{opts["core"]["dir_values"]}/{msp}/hlf-couchdb/cdb-{release}.yaml'
+            if not upgrade:
+                extra_vars = helm_extra_vars(version=version, config_yaml=config_yaml)
+                helm_install(
+                    opts["core"]["chart_repo"],
+                    "hlf-couchdb",
+                    f"cdb-{release}",
                     peer_namespace,
-                    f"cdb-{release}-hlf-couchdb",
-                    "COUCHDB_USERNAME",
```

Comment:
I'm guessing we have updated the Helm charts?

i.e. why are we removing the  "COUCHDB_USERNAME" and "couchdbUsername"? Are these autogenerated?